### ISSUE TITLE
Suspicious exports from the angular2/angular2

### ISSUE THREAD

**Issue by pkozlowski-opensource** [angular/angular#5700]
The current [angular2/angular2]() super-barrel mostly re-exports symbols from other barrels (core, common etc.). But there are few suspicious exports in there that don't seem to be exported from other main barrels. Those are:

```javascript
export * from './src/platform/dom/dom_adapter';
export * from './src/platform/dom/events/event_manager';
```

and 

```javascript
export {UrlResolver, AppRootUrl} from './compiler';
```

Why those exports are part of angular2/angular2? To which barrel they belong? We need to answer this question before we can kill the angular2/angular2 barrel.

**Comment by vsavkin** [comment]
>> export * from './src/platform/dom/dom_adapter'; export * from './src/platform/dom/events/event_manager';

Remove these and include platform/common_dom instead.

**Comment by vsavkin** [comment]
>> export {UrlResolver, AppRootUrl} from './compiler';

I actually do not know what to do about these. I'd say remove it. If anyone needs it, we should include the whole compiler instead.

**Comment by pkozlowski-opensource** [comment]
Fixed via 8c69497d8ed4602819508fa4b890ab0eb259aa19
